deep-learning/_posts/2018-08-13-FunctionalMapoftheWorld.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: review
+title:  "Functional Map of the World"
+tags:   deep-learning, classification, bounding-boxes, CNN, LSTM, remote-sensing, dataset
+author: Charles Authier
+pdf:  https://arxiv.org/pdf/1711.07846
+cite:
+  authors: "Gordon Christie, Neil Fendley, James Wilson, Ryan Mukherjee"
+  title:   "Functional Map of the World"
+  venue:   "CVPR 2018"
+---
+
+## Dataset
+Over 1 million images from over 200 countries. Each image, provide at least one bounding box annotation containing one of 63 categories.
+
+the dataset include:
+- UTM Zone (zone of projection)
+- Timestamp (year, month, day, hour, minute, second, and day of the week UTC)
+- GSD (Ground sample distance)
+- Angles (the sensor is imaging the ground, as well as the angular location)
+- Image+box sizes
+
+3 types of images: 8/4/3 bands, 4 and 8 are RGB with multispectral images (MSI) all images are 256x256 with roughly 30cm resolution.
```

Comment:
8 / 4 / 3 bands, 4 and 8 are RGB with multispectral images (MSI).
All images are 256x256 with a resolution of roughly 30 cm